zuul.d/project-templates.yaml

```diff
@@ -775,6 +775,29 @@
       queue: integrated
       jobs: *ansible-collections-ansible-netcommon-units-jobs
 
+- project-template:
+    name: ansible-collections-ansible-network-units
+    check:
+      jobs: &ansible-collections-ansible-network-units-jobs
+        - ansible-changelog-fragment
+
+        - build-ansible-collection:
+            required-projects:
+              - name: github.com/ansible-collections/ansible.netcommon
```

Comment:
doh, this should also have ansible-collection/ansible.network too.  Sorry

Reply:
updated. Thanks